server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestErrorsInDifferentFiles.java

```diff
@@ -101,20 +103,28 @@ void test() {
                     "Errors inside the copybook",
                     Error,
                     ErrorSource.COPYBOOK.getText()))
-            .put(
-                "areaA22",
+//            .put(
+//                "areaA22",
+//                new Diagnostic(
+//                    new Range(),
+//                    "Encountered invalid token. Analysis skipped to the next verb or period.",
+//                    Error,
+//                    ErrorSource.PARSING.getText()))
+            .put("waria",
                 new Diagnostic(
                     new Range(),
-                    "Encountered invalid token. Analysis skipped to the next verb or period.",
-                    Error,
+                    "The following token must start in Area A: CHILD1",
+                    Warning,
                     ErrorSource.PARSING.getText()))
             .put(
                 "dot",
                 new Diagnostic(
                     new Range(),
-                    "A period was assumed before \"PIC\".",
-                        Error,
-                    ErrorSource.PARSING.getText()))
+//                    "A period was assumed before \"PIC\".",
+                      "Syntax error on 'PIC'",
```

Comment:
the new message seems fine.
```suggestion
                      "Syntax error on 'PIC'",
```